docker-compose.yml

```diff
@@ -7,8 +7,8 @@ xlang:
         - yarpc-node
     environment:
         - XLANG_CLIENTS=yarpc-go,yarpc-java,yarpc-python,yarpc-node
-        - XLANG_SERVERS=yarpc-go,yarpc-java,yarpc-python,yarpc-node
-        - XLANG_BEHAVIORS=call,not_found
+        - XLANG_DIMENSION_SERVER=yarpc-go,yarpc-java,yarpc-python,yarpc-node
+        - XLANG_DIMENSION_BEHAVIOR=call,not_found
```

Comment:
@kriskowal note that now you need to specify `XLANG_DIMENSION_*` for every arg/dimension you want passed to `XLANG_CLIENTS`